libraries.json

```
@@ -13,53 +19,212 @@
+		"description": "RxJS powered state management for Angular applications, inspired by Redux",
+		"language": "JavaScript"
+	},
+	"NuxtJS": {
+		"imports": ["nuxtjs"],
+		"technologies": ["Web Development", "SPA", "Server side rendering"],
+		"description": "Nuxt.js is a free and open source web application framework based on Vue.js, Node.js, Webpack and Babel.js.",
+		"language": "JavaScript"
+	},
+	"Parcel": {
+		"imports": ["parcel"],
+		"technologies": ["Build tools"],
+		"description": "Blazing fast, zero configuration web application bundler.",
+		"language": "JavaScript"
+	},
+	"Proton Native": {
+		"imports": ["proton-native"],
+		"technologies": ["Desktop applications"],
+		"description": "Create native desktop applications through a React syntax, on all platforms",
+		"language": "JavaScript"
+	},
 	"ReactJS": {
 		"imports": ["react"],
-		"technologies": ["Web Development", "SPA"],
+		"technologies": ["Web Development", "SPA", "JavaScript Frameworks"],
 		"description": "A JavaScript library for building user interfaces",
 		"image": "https://cdn4.iconfinder.com/data/icons/logos-3/600/React.js_logo-512.png",
 		"language": "JavaScript"
 	},
-	"Svelte": {
+	"React Native": {
+		"imports": ["react-native"],
+		"technologies": ["Mobile applications"],
+		"description": "React Native combines the best parts of native development with React, a best-in-class JavaScript library for building user interfaces.",
+		"language": "JavaScript"
+	},
+	"Redux": { 
+		"imports": ["redux"],
+		"technologies": ["Web Development"],
+		"description": "A predictable state container for JavaScript apps.",
+		"image": "????",
+		"language": "JavaScript"
+	},
+	"Rollup": {
+		"imports": ["rollup"],
+		"technologies": ["Build tools"],
+		"description": "Parcel is a web application bundler, differentiated by its developer experience. It offers blazing fast performance utilizing multicore processing, and requires zero configuration.",
+		"language": "JavaScript"
+	},
+	"RxJS": {
+		"imports": ["rxjs"],
+		"technologies": ["Web Development"],
+		"description": "Reactive Extensions Library for JavaScript",
+		"image": "????",
+		"language": "JavaScript"
+	},
+		"Svelte": {
 		"imports": ["svelte"],
 		"technologies": ["Web Development", "SPA"],
 		"description": "Svelte is a radical new approach to building user interfaces. Whereas traditional frameworks like React and Vue do the bulk of their work in the browser, Svelte shifts that work into a compile step that happens when you build your app.",
 		"image": "https://raw.githubusercontent.com/sveltejs/branding/master/svelte-logo.png",
 		"language": "JavaScript"
 	},
+	"ThreeJS": {
+		"imports": ["three"],
+		"technologies": ["WebGL"],
+		"description": "DescriptionThree.js is a cross-browser JavaScript library and Application Programming Interface used to create and display animated 3D computer graphics in a web browser.",
+		"language": "JavaScript"
+	},
+	"Universal": {
```

```suggestion
	"Angular Universal": {
```

I think this is called `Angular Universal` not just `Universal`, not a 100% sure though. In the docs (https://angular.io/guide/universal) it says `This guide describes **Angular Universal**, a technology that renders Angular applications on the server.` (but it still calls it `Universal` after that)